src/api/services/adminServiceClient.ts

```diff
@@ -81,11 +81,38 @@ export const getSite = async (siteId: number): Promise<SiteDTO> => {
   return response.data;
 };
 
-export const getApiKeys = async (siteId: number): Promise<ApiKeyAdmin[]> => {
+export const getApiKeysFromAdmin = async (siteId: number): Promise<ApiKeyAdmin[]> => {
```

Comment:
I think the `FromAdmin` might be a bit redundant. What about `getApiKeysBySite`?

Reply:
Yeah that makes sense to me.